content/source/docs/enterprise-beta/registry/publish.html.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: "enterprise2"
+page_title: "Publishing Private Modules - Private Module Registry - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-registry-publish"
+---
+
+[vcs]: ../vcs/index.html
+
+# Publishing Modules to the Terraform Enterprise Private Module Registry
+
+~> **Note:** Currently, the private module registry is only enabled on private installs of TFE using Bitbucket Server as the VCS provider. Support for the SaaS version of TFE and other VCS providers is coming in a future update.
+
+TFE's private module registry lets you publish Terraform modules to be consumed by users across your organization. It works much like the public [Terraform Registry](/docs/registry/index.html), except that it uses your configured [VCS integrations][vcs] instead of requiring public GitHub repositories.
```

Comment:
Should we spell out `Terraform Enterprise (TFE)` on each page if we are going to abbreviate it?